components/ProgressView/src/MDCProgressView.h

```diff
@@ -86,4 +86,20 @@ IB_DESIGNABLE
          animated:(BOOL)animated
        completion:(void (^__nullable)(BOOL finished))completion;
 
+/**
+ Start the progress bar's indeterminate animation.
+ */
+- (void)startAnimating;
+
+/**
+ Stop the progress bar's indeterminate animation.
+ */
+- (void)stopAnimating;
+
+/**
+ Sets the progress bar to determinate and does a smooth animation between the indeterminate state
+ and the progress state using the current value for progress.
+ */
+- (void)moveToDeterminateSmoothly;
```

Comment:
Please make this API animatable (or not):

`- (void)setDeterminateMode:(BOOL) animated:(BOOL) completion:(**block**)`